app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```

Comment:
Why? Isn't this the default in ruby 2.x?

Reply:
This might be an editor artifact from me. I'll remove it.